framework/base/Application.php

```diff
@@ -615,6 +616,16 @@ public function getSecurity()
     }
 
     /**
+     * Returns the profiler component.
+     * @return \yii\base\Profiler the profiler application component.
+     * @since 2.0.12
+     */
+    public function getProfiler()
+    {
+        return $this->get('profiler');
```

Comment:
`profiler` should not belong to `Application` - it should be hold as static field at `Yii` class in the same way as `Logger`.